### PR TITLE
[SC-21322] ACP local development setup.

### PR DESCRIPTION
**Pull request by pushokwhite**
### Description

- https://spryker.atlassian.net/browse/SC-21322

#### Change log

- Added support for a new env variable `ACP_DOCKER_SDK_FILE` so that ACP location development can be set up.
- Adjusted the generation of the `deploy.bash` file to now add the value of `ACP_DOCKER_SDK_FILE` as the source if specified, also adding the additional AWS env variables for ACP Localstack.

### Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: generator/src/templates/deploy.bash.twig [hunk 1]
```diff
@@ -211,6 +216,14 @@ export TIDEWAYS_ENVIRONMENT_NAME=${TIDEWAYS_ENVIRONMENT_NAME}
 {# For avoid https://github.com/docker/compose/issues/9104 #}
 export DOCKER_COMPOSE_TTY_DISABLED="$(Environment::getDockerComposeTTY)"
 
+{% if _isAcpLocalDevelopmentEnabled %}
+# ACP
+export ACP_AWS_ENDPOINT={{ image['environment']['AWS_ENDPOINT'] | default('http://localhost:4566') }}
+export ACP_AWS_REGION={{ image['environment']['AWS_DEFAULT_REGION'] | default('eu-central-1') }}
+export ACP_AWS_ACCESS_KEY_ID={{ image['environment']['AWS_ACCESS_KEY_ID'] | default('test') }}
+export ACP_AWS_SECRET_ACCESS_KEY={{ image['environment']['AWS_SECRET_ACCESS_KEY'] | default('test') }}
```
Comment (matweew) [inline generator/src/templates/deploy.bash.twig:224]: is it not enough to have them in the `config-docker.dev.php`?

https://github.com/spryker/suite-nonsplit/blob/beta/app-store/develop/config/Shared/common/config_message-broker-development.php#L13

File: generator/src/templates/deploy.bash.twig [hunk 2]
```diff
@@ -83,6 +83,11 @@ import service/database/{{ services['database']['engine'] }}.sh
 import service/broker/{{ services['broker']['engine'] }}.sh
 import service/scheduler/{{ services['scheduler']['engine'] }}.sh
 
+{% if _isAcpLocalDevelopmentEnabled %}
+# ACP
+[[ -f "{{ image['environment']['ACP_DOCKER_SDK_FILE'] }}" ]] && source "{{ image['environment']['ACP_DOCKER_SDK_FILE'] }}"
```
Comment (matweew) [inline generator/src/templates/deploy.bash.twig:88]: @alexanderM91  what do you think about using env vars in such way https://github.com/spryker/suite-nonsplit/pull/6810/files#diff-fd2fd949d614678c890a3684410e8127ad10428d1625b047267f971941735c89R23 ?

File: generator/src/templates/deploy.bash.twig (hunk 1 above)
Comment (pushokwhite) [reply generator/src/templates/deploy.bash.twig:224]: They are used by `bash` not `php` here.

**Review by alexanderM91 — APPROVED** [review]

**Comment by pushokwhite** [comment]
The latest master is merged already.

**Pull request merged**